atlp-rwanda/code-ninjas-be
# #181414764 feature (trip request) add trip request functionality 181414764

### What does this PR do?
 Users should be able to make trip request
### Description of Task to be completed?
The requestor should be able to enter Where he is leaving from, Where he is going to, date of travel, date of return, travel reasons, accommodation

### How should this be manually tested?
Get started By Running commands bellow:
  - clone repo
  - switch to [[ft-trip-request-181414764](https://github.com/atlp-rwanda/code-ninjas-be/tree/ft-trip-request-181414764)]) branch
  - ``yarn install``: to install all dependencies
  - Create .env file
  - Set up .env as stated in .env example
  - run `yarn run  dev` to start the server
  - Open your browser and open on this route `/login` and choose between Facebook and google
  - After successful logging in  you will be able to see this message `{"Message":"logged in successfully"}`

### Any background context you want to provide?
### What are the relevant pivotal tracker stories?
[#181414764](https://www.pivotaltracker.com/story/show/181414764)
